product_equivalent_category/views/product_equivalent_category_views.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="product_equivalent_category_search_view" model="ir.ui.view">
+        <field name="name">product.equivalent.category.search</field>
+        <field name="model">product.equivalent.category</field>
+        <field name="arch" type="xml">
+            <search string="Product Equivalent Categories">
+                <field name="name" string="Product Equivalent Categories"/>
+            </search>
+        </field>
+    </record>
+    <record id="product_equivalent_category_form_view" model="ir.ui.view">
+        <field name="name">product.equivalent.category.form</field>
+        <field name="model">product.equivalent.category</field>
+        <field name="arch" type="xml">
+            <form class="oe_form_configuration">
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button class="oe_stat_button"
+                            name="%(product.product_template_action_all)d"
```

Comment:
The button accurately shows how many products there are in this equivalent category. However, clicking on it shows all products. Is this intentional ?
![Screenshot from 2020-02-21 14-43-58](https://user-images.githubusercontent.com/19902174/75039504-208cc200-54b9-11ea-92a8-79fd6939de7a.png)

Reply:
Fixed :+1: